chia/_tests/pools/test_pool_rpc.py

```diff
@@ -1020,3 +1022,48 @@ async def status_is_leaving_no_blocks() -> bool:
 
         # Eventually, leaves pool
         assert await status_is_farming_to_pool()
+
+    @pytest.mark.anyio
```

Comment:
Is there a reason this isn't `WalletTestFramework`?

Reply:
nothing in this file was using WalletTestFramework at the time, so I followed the conventions here. But I think your question is a good one and I'll look to convert it to use the newer concepts